app/views/digital_service_info.scala.html

```diff
@@ -14,16 +14,17 @@
  * limitations under the License.
  *@
 
-@import uk.gov.hmrc.cataloguefrontend.RepoType.RepoType
 @import uk.gov.hmrc.cataloguefrontend.UserManagementConnector.{NoData, UMPError}
-@import uk.gov.hmrc.cataloguefrontend.{DigitalServiceDetails, RepoType, ViewMessages}
-@import uk.gov.hmrc.cataloguefrontend.UserManagementConnector.TeamMember
-@import uk.gov.hmrc.cataloguefrontend.ViewMessages._
+@import uk.gov.hmrc.cataloguefrontend.{DigitalServiceDetails, ViewMessages}
+@import uk.gov.hmrc.cataloguefrontend.connector.RepoType
+@import uk.gov.hmrc.cataloguefrontend.connector.RepoType.RepoType
 @import uk.gov.hmrc.cataloguefrontend.DisplayableTeamMember
 @import uk.gov.hmrc.cataloguefrontend.actions.UmpVerifiedRequest
 
-@(timestampedRepos: DigitalServiceDetails,
-  digitalServiceOwner: Option[DisplayableTeamMember]
+@(
```

Comment:
My subjective preference when talking about template refactoring would be to use injection for things like ViewMessages. The benefit of that would be that it takes that necessity of injecting things needed by a template to a controller which effectively does not need it. I believe such approach makes things cleaner and easier to reason about. What do you think?